scripts/stage1.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+function die
+{
+	echo "$1" >&2
+	exit 1
+}
+
+[[ $# -eq 1 ]] || die "Illegal number of parameters"
+target="$1"
+workdir=$(mktemp -d)
+img=$workdir/img
+base="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+#
+# Set up working directory
+#
+cd $workdir
+sudo mkdir img
```

Comment:
Is `sudo` required here and in the calls below? If so, can you comment on why we need it.. if not, can you remove them..

IMO, we should (ideally) be able to build the package (based on my understanding, this is only called during the package build, is that correct?) without needing root privileges.. if we do need root, I'd like better understand why it's required.